pvlib/tests/ivtools/test_sdm.py

```diff
@@ -379,4 +380,23 @@ def test__calc_theta_phi_exact_vector():
     assert np.isnan(theta[0])
     assert np.isnan(theta[1])
     assert np.isnan(phi[0])
-    np.testing.assert_allclose(phi[1], 2.2079, atol=.0001)
+    assert_allclose(phi[1], 2.2079, atol=.0001)
+
+
+def test_pvsyst_temperature_coeff():
+    # test for consistency with dP/dT estimated with secant rule
+    params = {'alpha_sc': 0., 'gamma_ref': 1.1, 'mu_gamma': 0.,
+              'I_L_ref': 6., 'I_o_ref': 5.e-9, 'R_sh_ref': 200.,
+              'R_sh_0': 2000., 'R_s': 0.5, 'cells_in_series': 60}
+    expected = -0.004886706494879083
```

Comment:
```suggestion
    expected = -0.004887
```

Reply:
In the past @adriesse has gently nudged us towards using reasonable precision when testing expected. While it can be a pain, I do think it makes the tests more understandable and makes it easier to catch issues like the one you fixed in the last commit.

Reply:
I don't object on principle to the suggestion. But here I think more precision is useful, in case the test goes wonky in the future, and someone is reconstructing the expected value to know they have that calculation correct.

Reply:
I only vaguely recall the nudging occasion, but I have a general pet peeve about people producing too many digits of noise (e.g. NSRDB).  You can't even compress it!  

For tests I think it depends on what the test is aiming to accomplish.  In this case you might see what happens if someone passes in float32 or maybe even float16.  I often use float32.

Looking in the code now it seems like lots of potential changes/improvements in the functions that are called could affect the numeric result without breaking the fundamental calculation.  Do you want that detected?

One thing you could perhaps consider would be to not rely on defaults when calling `derivative`.  Or do away with `derivative` and call `pmax` twice yourself.  Then the user doesn't have to go to scipy documentation to figure out what is actually being done.

Reply:
Good points about the options in `scipy.misc.derivative`. I didn't see an advantage to explicitly choosing options, or alternatively, hardcoding a particular derivative rule, but I'm open to doing either.

Peeve acknowledged and shared. The precision for `expected` is to enable reconstructing this value and being confident that the reconstruction is the same as the test originator's calculation.

I wasn't thinking about detecting small deviations in the output of `pvsyst_temperature_coeff`, only producing a value that is "good enough" in some sense. That's why I set `rtol=0.0005`; I think 4 digits agreement is enough to say two temperature coefficients are the same.

Reply:
Sorry, I hadn't noticed the rtol.  I got mislead by all those digit thinking the must be there because you  wanted them to be there.

Reply:
@cwhanse do you want to make revisions or go ahead and merge? I agree that @adriesse makes good points, but I am also fine with saying go read the scipy documentation if you want that level of detail.

Reply:
I'm OK merging this and improving it later, if issues arise. The only options are point spacing and order of the derivative rule, and the defaults (dx=1.0C, order=3) are reasonable for this case.